boa_engine/src/bytecompiler.rs

```diff
@@ -559,6 +559,7 @@ impl<'b> ByteCompiler<'b> {
                     Const::Bool(false) => self.emit(Opcode::PushFalse, &[]),
                     Const::Null => self.emit(Opcode::PushNull, &[]),
                     Const::Undefined => self.emit(Opcode::PushUndefined, &[]),
+                    Const::Elision => unreachable!(),
```

Comment:
Do you think it makes sense to use `Node::Empty` as the elision node in the arraydecl to avoid this?

Reply:
> Do you think it makes sense to use `Node::Empty` as the elision node in the arraydecl to avoid this?

Hmmm... Yeah, `Node::Empty` is a far better choice.